File: locuslabs-android-sdk/examples/RecommendedImplementation/app/src/main/java/com/example/locuslabs/recommendedimplementation/FlightCardActivity.java
package com.example.locuslabs.recommendedimplementation;

import android.app.Activity;
import android.os.Bundle;

public class FlightCardActivity extends Activity {
    private static final String TAG = "FlightCardActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // Sets the content view to match ./res/layout/flight_card.xml
        setContentView(R.layout.flight_card);
    }
}

File: locuslabs-android-sdk/examples/RecommendedImplementationKotlin/app/src/main/java/com/example/locuslabs/recommendedimplementation/FlightCardActivity.kt
package com.example.locuslabs.recommendedimplementation

import android.app.Activity
import android.os.Bundle

class FlightCardActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // Sets the content view to match ./res/layout/flight_card.xml
        setContentView(R.layout.flight_card)
    }

    companion object {
        private val TAG = "FlightCardActivity"
    }
}
